web-api/src/applicationContext.js

```diff
@@ -1285,7 +1289,7 @@ const environment = {
   elasticsearchEndpoint:
     process.env.ELASTICSEARCH_ENDPOINT || 'http://localhost:9200',
   masterDynamoDbEndpoint:
-    process.env.MASTER_DYNAMODB_ENDPOINT || 'dynamodb.us-east-1.amazonaws.com',
+    process.env.MASTER_DYNAMODB_ENDPOINT || 'http://localhost:8000',
```

Comment:
Do we need to add this to the environment variables in AWS Secrets Manager? Or is it derived and set elsewhere?

Reply:
I believe this was just incorrectly defaulted, and should have been the same as DYNAMODB_ENDPOINT @jamesobrooks